data/jsons/dependencies.json

```diff
@@ -126,12 +126,14 @@
 	{
 	    "dependency": "chrpath",
 	    "type": "exec",
-	    "exec": "chrpath"
+	    "exec": "chrpath",
+            "required": "true"
```

Comment:
would I need jsonschema module to use the json schema? if so it doesn't make sense since this scripts is run before anything and to check even for jsonschema presence.